packages/common/src/intl/locales/en/distribution.json

```diff
@@ -98,7 +97,7 @@
   "messages.deleted-requisitions": "Deleted {{number}} requisitions",
   "messages.error-saving-shipment": "Error saving shipment 🥺",
   "messages.must-allocate-all-lines": "Cannot change the status until all lines have been allocated.",
-  "messages.select-rows-to-delete": "Select rows to delete",
+  "messages.select-rows-to-delete-them": "Select rows to delete them",
```

Comment:
First I though its reads not very well, e.g. compared to "Select rows for deletion" or "Select rows to be deleted" but your version is actually very clear and unambiguous :-)